<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CriarModulo</class>
 <widget class="QWidget" name="CriarModulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criar Modulo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QPushButton" name="btnCriarModulo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Criar MÃ³dulo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditNomeModulo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(234, 234, 234)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditObsModulo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(234, 234, 234)</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandButtonVoltar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton
{
	border: none;
}

QCommandLinkButton:hover
{
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../setavoltar.ico</normaloff>../setavoltar.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Obs:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Carro</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Littleendian">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Litlle-endian</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Bigendian">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Big-Endian</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomeCarro">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INDEFINIDO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
